packages/atomic/package.json

```diff
@@ -25,29 +25,31 @@
   ],
   "scripts": {
     "clean": "rimraf -rf dist/*",
-    "build": "npm run build:locales && npm run build:stencil && npm run doc:list-assets && npm run build:storybook && npm run validate:definitions",
```

Comment:
`doc:list-assets` is only used by Storybook, so I moved it into `build:storybook`.